edge/docs/installation/inception_internals.md

```diff
@@ -0,0 +1,122 @@
+>The final directory name in the path must be `vision-edge`. EG `/opt/ibm/vision-edge`.
+
+3. User logs in to the Docker repository.
+
+   1. An IBM Cloud Entitlement Key is required. This can be obtained by logging in to https://myibm.ibm.com/products-services/containerlibrary and copying the entitlement key.
+   2. Login to the docker repository
+      ```
+      docker login cp.icr.io --username cp --password <entitlement key>
+      ```
+      > the user name must be `cp` and the `<entitlement key>` is the one copied in step 3.1 above
+
+4. User `cd`'s to the `<install root>` and executes the `docker run` command:
+    ```
+    cd <install root>
+
+    docker run --rm -v `pwd`:/opt/ibm/vision-edge --privileged -u root cp.icr.io/cp/visualinspection/vision-edge-inception:8.3.0
+    ```
+    - This will pull the inception image and run it.
+    - The inception container will 
+      - create the required sub-directories under the \<install root> with the required ownership and permissions.
+        > The userid and groupid are the ones that are used internally by all the MVIE Service containers. These will be 900:9999 in the host OS. Within the containers the user and groupnames are both `visionedge`
+      - populate sub-directories with scripts, templates, and properties files
+
+5. User runs the startedge.sh script
+    ```
+    ./startedge.sh
+    ```
+    - This will:
+      - display the license agreement, which the user must accept to continue
+        > NOTE on Macs, the `locale` command does not provide the locale name, which will cause the license acceptance script to fail. This can be worked around by commenting out the line `source $HOST_SERVICE_ROOT/bin/license.sh` in the startedge.sh script.
+      - pull the 3 MVIE service images
+      - generate certificates for the server
+      - initialize the database
+      - start the 3 service containers:
+        - vision-edge-controller
+        - vision-edge-cme
+        - vision-edge-dle
+> **NOTE** The startedge.sh script can be run at any time to restart the MVIE service containers. This is required whenever changes to the default configuration are made.
+
+The system is now up and running. 
+
+The startedge.sh script will display the URL to access the Web UI. To login, the default userid and password are `masadmin` and `VisionP@ssw0rd`.
+
+## Operational Considerations
+**Pre-Install**
+- Storage configuration
+  - When run at the full capacity, an MVIE system will produce significant volumes of image files and metadata. This volume will be determined by the compute resources available on the system so there is no single storage configuration that can be recommended for all deployments, but the following are general recommendations for storage layout:
+    - The MVIE \<install root> should be located on a different storage volume than the one that docker will use. EG docker typically stores things under /var so the MVIE root should be located in a different volume.
```

Comment:
quote '\<install root>' and '/var'.
Also just noticed the difference in slashes. Is that intentional?

Reply:
> quote '<install root>' and '/var'.
> Also just noticed the difference in slashes. Is that intentional?

Yeah I guess backquotes would work here. The backslash in front of install root is escaping the `<` so it renders correctly but backquoting will be better.